NetworkService/Sources/NetworkService/NetworkService.swift

```diff
@@ -0,0 +1,8 @@
+final class NetworkService {
+    /// Stub method to check possibility to use this NetworkService in your App.
+    ///
+    ///  - Returns: true
+    func use() -> Bool {
```

Comment:
nit:
```suggestion
    func testUse() -> Bool {
```

or

```suggestion
    func dummyMethod() -> Bool {
```

to make sure everyone understands this is a test method from its name.

Reply:
fixed